auth0/src/test/java/com/auth0/android/util/OkHttpTls12CompatTest.java

```diff
@@ -0,0 +1,75 @@
+package com.auth0.android.util;
+
+import android.app.Activity;
+
+import com.squareup.okhttp.ConnectionSpec;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.TlsVersion;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.List;
+
+import javax.net.ssl.SSLSocketFactory;
+
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = com.auth0.android.auth0.BuildConfig.class, sdk = 21, manifest = Config.NONE)
+public class OkHttpTls12CompatTest {
+
+    Activity activity;
+    @Mock OkHttpClient client;
+
+    @Before
+    public void setUp(){
+        MockitoAnnotations.initMocks(this);
+        activity = Robolectric.setupActivity(Activity.class);
+    }
+
+    @Test
+    @Config(sdk=22)
+    public void shouldNotConfigTlsPostApi21() {
```

Comment:
Because the min API for this library is 15 and that's excluded from the TLS logic, please add a test for that one as well.

Reply:
I tried but Robolectric doesn't support API level 15 so we can't just add that like other supported APIs.